lib/individual/health.dart

```diff
@@ -1,182 +1,220 @@
+                                  padding:
+                                      EdgeInsets.fromLTRB(15.w, 0, 15.w, 0),
+                                  child: SearchField(
+                                    hint: "Enter Organization's name",
+                                    suggestions: const [
+                                      'organization 1',
+                                      'organization 2',
+                                      'organization 3',
+                                      'organization 4',
+                                      'organization 5',
+                                      'organization 6',
+                                    ],
+                                    searchInputDecoration: InputDecoration(
+                                      enabledBorder: OutlineInputBorder(
+                                        borderSide: BorderSide(
+                                            color: Colors.grey, width: 2.w),
+                                        borderRadius:
+                                            BorderRadius.circular(80.r),
+                                      ),
+                                      focusedBorder: OutlineInputBorder(
+                                        borderRadius:
+                                            BorderRadius.circular(80.r),
+                                        borderSide: BorderSide(
+                                            color: const Color.fromARGB(
+                                                255, 137, 184, 189),
+                                            width: 2.w),
+                                      ),
+                                      fillColor: const Color.fromARGB(
+                                          255, 186, 231, 235),
+                                      filled: true,
+                                      suffixIcon: const Icon(
+                                        Icons.arrow_drop_down,
+                                        color: Colors.grey,
+                                        size: 40,
+                                      ),
                                     ),
                                   ),
                                 ),
-                              ),
-                            ],
-                          )
-                        ]),
-                  ),
-                ],
-                // ]
-              ),
-            ),
-            // ],
-          ),
-        ),
-      ),
-    );
-  }
-}
+
+                              ],
+
+                                )
+                              ]),
+
+
+                              
+                         )
+
+
+
+                          ]
+                           // ]
+                        ),
+                        )
+                      // ],
+                      
+                    )
+                    ),
+              )
+            );
```

Comment:
The brackets are mismatched at the end of the file. I believe there are 2 '}' missing at the end. Kindly make sure to correct this in your fix commit.